app/src/test/java/com/codesoom/assignment/application/UserServiceTest.java

```diff
@@ -0,0 +1,145 @@
+
+@SpringBootTest
+@DisplayName("UserService 클래스")
+class UserServiceTest {
+
+    @Autowired
+    private UserService userService;
+
+    List<UserData> testUserDatas = new ArrayList<>();
+
+    @BeforeEach
+    void setUp() {
+        testUserDatas.add(UserData.builder()
+                .name("Hyuk")
+                .password("!234")
+                .email("pjh0819@naver.com")
+                .build());
+
+        testUserDatas.add(UserData.builder()
+                .name("Update Hyuk")
+                .password("123$")
+                .email("pjh9999@naver.com")
+                .build());
+    }
+
+    @Nested
+    @DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+    class createUser_메소드는 {
+
+        @Nested
+        @DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+        class 등록할_User가_주어진다면 {
+
+            UserData givenUserData;
+
+            @BeforeEach
+            void prepare() {
+                givenUserData = testUserDatas.get(0);
+            }
+
+            @Test
+            @DisplayName("User 객체를 생성한다")
+            void User를_생성하고_리턴한다() {
+                User user = userService.createUser(givenUserData);
+
+                User foundUser = userService.getUser(user.getId());
+
+                assertThat(foundUser.getName()).isEqualTo(givenUserData.getName());
+                assertThat(foundUser.getEmail()).isEqualTo(givenUserData.getEmail());
+            }
+        }
+    }
+
+    @Nested
+    @DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+    class updateUser_메소드는 {
+
+        @Nested
+        @DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+        class 등록된_User의_id와_수정할_User가_주어진다면 {
+
+            UserData givenUserData;
+            Long givenId;
+
+            @BeforeEach
+            void prepaer() {
+                User user = userService.createUser(testUserDatas.get(0));
+                givenId = user.getId();
+                givenUserData = testUserDatas.get(1);
+            }
+
+            @Test
+            void 해당_id의_User를_수정하고_리턴한다() {
+                userService.updateUser(givenId, givenUserData);
+
+                User foundUser = userService.getUser(givenId);
+
+                assertThat(foundUser).isNotNull();
+                assertThat(foundUser.getName()).isEqualTo(givenUserData.getName());
+            }
+        }
+
+        @Nested
+        @DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+        class 등록되지_User의_id와_수정할_User가_주어진다면 {
+
+            UserData givenUserData;
+            Long givenInvalidId = 9999L;
+
+            @BeforeEach
+            void prepaer() {
+                givenUserData = testUserDatas.get(1);
+            }
+
+            @Test
+            void 해당_id의_User를_수정하고_리턴한다() {
+                assertThatThrownBy(() -> userService.updateUser(givenInvalidId, givenUserData)).isInstanceOf(UserNotFoundException.class);
```

Comment:
한 줄이 너무 길어요. 적절히 개행해서 리뷰어(동료들)가 읽기 좋게 작업해주세요 ㅎㅎ
만약 잘 모르겠다면 한 줄에 `.`이 하나씩 오도록 개행해 주면 됩니다.
```suggestion
                assertThatThrownBy(() -> userService.updateUser(givenInvalidId, givenUserData))
                    .isInstanceOf(UserNotFoundException.class);
```